monetio/models/_cesm_fv_mm.py

```diff
@@ -116,3 +157,98 @@ def _ensure_mfdataset_filenames(fname):
     if len(netcdfs) >= 1:
         netcdf = True
     return names, netcdf
+
+
+def _calc_pressure(dset):
+    """Calculates midlayer pressure using P0, PS, hyam, hybm
+    Parameters
```

Comment:
The headings need a blank line above them.

Reply:
This hasn't been address yet.

Reply:
Please add a blank line above "Parameters" and "Returns". And same in your new hydrostatic height function.